classes/PodsInit.php

```diff
@@ -469,6 +469,10 @@ public function core() {
 		if ( method_exists( $avatar, 'get_avatar' ) ) {
 			add_filter( 'get_avatar', array( $avatar, 'get_avatar' ), 10, 4 );
 		}
+
+		if ( method_exists( $avatar, 'get_avatar_data' ) ) {
+			add_filter( 'get_avatar_data', array( $avatar, 'get_avatar_data' ), 10, 2 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: bdb199c013885b1de31d8c7b30e432b6</sub>